modelscope/pipelines/audio/ans_pipeline.py

```diff
@@ -48,7 +48,7 @@ def preprocess(self, inputs: Input, **preprocess_params) -> Dict[str, Any]:
         if len(data1.shape) > 1:
             data1 = data1[:, 0]
         if fs != self.SAMPLE_RATE:
-            data1 = librosa.resample(data1, fs, self.SAMPLE_RATE)
+            data1 = librosa.resample(data1, orig_sr=fs, target_sr=self.SAMPLE_RATE)
```

Comment:
lint test failed, please refer to https://github.com/modelscope/modelscope/blob/master/docs/source/develop.md#1-code-style to format your code